nms/packages/magmalte/generated/MagmaAPIBindings.js

```diff
@@ -190,6 +190,8 @@ export type ci_node = {
 export type config_info = {
```

Comment:
This file should be modified *if and only if* the swagger files are modified, as it is generated from the swagger files. Either include the swagger file changes in this pull request, or include the changes to file in the pull request where you have changed the swagger files.

Reply:
The corresponding swagger and proto changes along with orc8r and AGW are in PR #9161